libbeat/processors/add_nomad_metadata/nomad.go

```diff
@@ -0,0 +1,166 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
Could you please rename this file to `add_nomad_metadata.go`? Just to follow the conventions of other `add_*_metadata` processors.

Reply:
Done